randovania/games/cave_story/logic_database/Sand Zone.txt

```diff
@@ -203,10 +203,7 @@ Extra - starting_script: <TRA0010:0099:0036:0033
                       Any of the following:
                           Missiles ≥ 11 and Kill Bosses with Missiles (Beginner)
                           Missiles ≥ 10 and Kill Bosses with Missiles (Intermediate)
-              Any of the following:
-                  # Energy Requirements
-                  Combat (Intermediate) or Normal Damage ≥ 19
-                  Combat (Beginner) and Normal Damage ≥ 10
+              Trivial
```

Comment:
toss a `FIXME: energy requirements here cause generation failures` comment on

Reply:
i have a TODO, I'll change it to a fixme

the pretty-print that generates the txt files doesnt show it, due to it being trivial :(

Reply:
in that case make it a 1 damage req for now. idc about the edge case of presets starting with 1hp

Reply:
for some reason there's now 1 resolver failure :(

Reply:
Nevermind, it seems i didnt look through things correctly, and this is caused by something else.

So this PR bumps generation from 96/100 to 99/100.

Currently trying to figure out what the last failure is caused by

Reply:
Found it, it was called toroko+.